package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "kuzzle",
   "author": "The Kuzzle Team <support@kuzzle.io>",
-  "version": "1.9.1",
+  "version": "1.9.2",
```

Comment:
The package-lock.json file must also be updated.

Reply:
:+1: